Presentation.Web/Tests/01_Parallel/It-Contract/LocalAdmin.ItContract.UiCustomization.e2e.spec.ts

```diff
@@ -88,17 +91,38 @@ describe("Local admin is able customize the IT-Contract UI", () => {
             .then(() => verifyTabVisibility(name, tabSref, false));   //Verify that the tab has now been hidden
     }
 
+    function testTabCustomizationWithSubtreeIsComplete(name: string, settingIds: Array<string>, tabSref: string) {
+        console.log("testTabCustomization for ", name, " and tabSref:", tabSref);
+
+        return verifyTabVisibility(name, tabSref, true)                                                                   //Check that the tab is visible before the change
+            .then(() => openUiConfiguration())                                                                            //Open Ui Customization page and show details
+            .then(() => settingIds.forEach((id) => { checkStateAndToggleSetting(id, false) }))                            //Toggle the setting
```

Comment:
this will be unreliable.. you need to do a reduce in stead and pass the previous promise as accumulant and chain it into the next. In that way you can create a sequence, and your test will be stable - also in varying load levels on the servers